torchbenchmark/score/configs/v1/config-v1.yaml

```diff
@@ -0,0 +1,124 @@
+benchmarks:
+  test_eval[BERT_pytorch-cpu-eager]:
+    norm: 0.05529004814998188
```

Comment:
What is a "norm" here?

Reply:
It is actually the mean latency of the test. The term "norm" is inherited from V0.

Reply:
It is just the mean latency of the test. The term `norm` is inherited from v0.